src/pages/receive-tokens/components/address-qr-code.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { memo } from 'react';
+import { createQR } from '@vkontakte/vk-qr';
+import { Box, color, Flex, FlexProps } from '@stacks/ui';
+
+export const QrCode: React.FC<{ principal: string } & FlexProps> = memo(
+  ({ principal, ...rest }) => {
+    const qrSvg = React.useMemo(
+      () =>
+        createQR(principal, {
+          ecc: 0,
+          qrSize: 180,
+          backgroundColor: color('text-body'),
+          foregroundColor: color('invert'),
+        }),
+      [principal]
+    );
+
+    const qr = <Box dangerouslySetInnerHTML={{ __html: qrSvg }} />; // Bad?
```

Comment:
I know this code was just moved, but any idea why we do this?

Reply:
I didn't pay attention to this. I guess this is the only way to put embed if the library only returns html/svg as a string?